### DISCUSSION TITLE
Restrict Organization owners from bypassing branch protection rule

### DISCUSSION THREAD

**Discussion by JoshuaAtEriks** [community/community#52530] [Enterprise]
Is it possible to enforce at enterprise level to restrict organisation owners by bypassing branch protection ?
Thanks

**Comment by queenofcorgis** [comment]
👋🏼 Thanks for your patience. It is not possible. Org admins are also repo admins. And by default, repo admins can bypass branch protections.

**Comment by patrick-knight** [comment]
Hey @JoshuaAtEriks, 

We recently introduced a beta of [Repository rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/managing-rulesets-for-a-repository#editing-a-ruleset) which allows for more granular control of bypass for both org and repo admins and does not bypass them by default. It is not currently available at the enterprise today but that functionality is on the roadmap.
